react/src/helper/index.tsx

```diff
@@ -171,7 +171,7 @@ export function iSizeToSize(
   const sizes = ['B', 'K', 'M', 'G', 'T', 'P', 'E'];
   const [sizeValue, sizeUnit] = parseUnit(sizeWithUnit);
   const sizeIndex = sizes.indexOf(sizeUnit.toUpperCase());
-  if (sizeIndex === -1 || isNaN(sizeValue)) {
+  if (sizeIndex === -1 || isNaN(sizeValue) || !sizeValue) {
```

Comment:
@agatha197, Can I ask why did you add the `!sizeValue` condition? This causes an error when `iSizeToSize("0g","b");` is called.